pkg/vc/models.go

```diff
@@ -17,6 +17,8 @@ const (
 	AuthorizationCredentialContext = "https://trustbloc.github.io/context/vc/authorization-credential-v1.jsonld"
 	// AuthorizationCredentialType is the JSON-LD type for the AuthorizationCredential.
 	AuthorizationCredentialType = "AuthorizationCredential"
+	// AuthorizationCredentialURIID is the non-fragment URI portion of the AuthorizationCredential JSON-LD context ID.
+	AuthorizationCredentialURIID = "https://example.org/examples"
```

Comment:
Why are we adding uri's from `example.org` in actual codebase ?

Reply:
@Moopli Filip can you answer this question? ^

Reply:
This URI is only used as part of the type identifier of the json-ld object - it's not dereferenced, but rather, used to identify the 'domain' of the object. 
We should change https://trustbloc.github.io/context/vc/authorization-credential-v1.jsonld to use a different URI for its IDs, instead of `https://example.org/examples`.